mpas_analysis/obs/observational_datasets.xml

```diff
@@ -254,35 +251,42 @@
       ocean
     </component>
     <description>
-     (missing)
+      The Trenbert and Caron oceanic meridional heat transport is computed
+      by computation of energy balance of the atmosphere, adjusted to fit
+      physical constraints, and using two reanalysis products: the National 
+      Centers for Environmental Prediction-National Center for Atmospheric 
+      Research (NCEP-NCAR) reanalysis and the European Centre for Medium-Range 
+      Weather Forecasts (ECMWF) product. The analysis focuses on the period 
+      from February 1985 to April 1989 when there are reliable top-of-the-atmosphere 
+      radiation data from the Earth Radiation Budget Experiment. 
     </description>
     <source>
-      (missing)
+      Data available upon request from Dr. Kevin Trenberth.
     </source>
     <releasePolicy>
-      (missing)
+      Acknowledgment: please cite: Trenberth and Caron (2001). Estimates of 
+      Meridional Atmosphere and Ocean Heat Transports, J. of Climate, 14, 3433-3443.
     </releasePolicy>
     <references>
-      [Trenberth and Caron (2000)](https://doi.org/10.1175/1520-0442%282000%29013&lt;4358%3ATSORSL&gt;2.0.CO%3B2)
+      [Trenberth and Caron (2001)](https://doi.org/10.1175/1520-0442(2001)014<3433:EOMAAO>2.0.CO;2)
```

Comment:
You have to escape a bunch of characters, unfortunately, that aren't allowed in XML (or html).  These include:`(` to `%28`, `)` to `%29`,  `<` to `&lt;`,  `>` to `&gt;`, `:` to `%3A` and `;` to `%3B`.  I know it's a huge pain, and it's good that DOIs stopped using these horrible characters after the early 2000s.